src/GraphQL/Utilities/Federation/AnyValue.cs

```diff
@@ -1,11 +1,16 @@
-using GraphQL.Language.AST;
+using GraphQLParser.AST;
 
 namespace GraphQL.Utilities.Federation
 {
-    public class AnyValue : ValueNode<object>
+    public class AnyValue : GraphQLValue
```

Comment:
I don't think there's any point to this class.  It's only created when printing a `AnyScalarGraphType` that contains a default value.  But there exists no actual code to support printing such a node.  I suggest removing it and throwing an exception within `AnyScalarGraphType.ToAST`.

If I understand federation correctly, `AnyScalarGraphType` would only be used for portions of a schema that is not handled by the active implementation, and as such, would not be used for input values at all (so it could not have a default value, so `AnyValue` is unused/unnecessary).

Reply:
And if we wanted to support `AnyScalarGraphType.ToAST` properly, the `ToAST` method should really turn the object back into a `GraphQLValue` (list/object/etc).  After all, that's the input format in `ParseLiteral`.  It would then actually be useful as a dynamic type of input/output object (which we have been asked for multiple times), not just for federation.

Reply:
Agreed, indeed. Removed.